build.gradle

```diff
@@ -64,6 +66,7 @@ application {
 modularity.patchModule("test", "fastparse_2.12-1.0.0.jar")
 modularity.patchModule("test2", "fastparse-utils_2.12-1.0.0.jar")
 modularity.patchModule("test3", "sourcecode_2.12-0.1.4.jar")
+modularity.patchModule("apollo.runtime", "apollo-api-jvm-2.5.8.jar")
```

Comment:
@koppor

Reply:
Please add a comment that if this is fixed, we can remove the `jar`